blogs/tekton-oadp-nonadmin/install_templates/templates/03-rbac-pipeline-role.yaml

```diff
@@ -17,9 +23,21 @@ objects:
     name: oadp-nonadmin-role
   rules:
   - apiGroups:
+    # By default restore resources will NOT be provided by this template.
+    # It is recommended that restores are executed by OpenShift Admin's in production environments.
+    # For demonstration purposes only you may uncomment the restore resources.
     - velero.io
     resources:
-    - '*'
+    - 'backuprepositories'
```

Comment:
Do we want these users to be able to create a new back repo they control them backup the whole cluster?

Reply:
backuprepositories are created per restic backup correct? This is not user defined iiuc and would be needed for any restic backup. Maybe I'm wrong

Reply:
@shawn-hurley Backup repositories are created by velero for restic/kopia/datamover backups. End users never create them (although for debugging purposes, sometimes they must remove them, but I'm assuming that can remain for admins only).

Reply:
But I don't think the non-admin user or tekton needs any permissions around this -- the velero pod will create it as needed.

Reply:
>backuprepositories are created per restic backup correct? 

Per backupStorageLocation.

Reply:
@kaovilai also per namespace.